R/CDPSession.R

```diff
@@ -317,18 +421,17 @@ CDPConnexion <- R6::R6Class(
 )
 
 chr_get_ws_addr <- function(host = "localhost", port = 9222, secure = FALSE) {
-  url <- build_url(host, port, secure)
   "!DEBUG Retrieving Chrome websocket entrypoint at http://localhost:`port`/json ..."
-  open_debuggers <- tryCatch(
-    jsonlite::read_json(paste0(url, "/json"), simplifyVector = TRUE),
-    error = function(e) list()
-  )
-
-  address <- open_debuggers$webSocketDebuggerUrl[open_debuggers$type == "page"]
+  targets <- list_targets(host, port, secure)
+  active_target <- purrr::detect(targets, ~ identical(.x$type, "page"))
```

Comment:
TBH, I do not understand the effect of `activate_target()`. My guess is that it corresponds to the selected tab, but I did not notice any difference using the activated tab and the other ones.
What difference have you noticed between them?

For now, I do not feel that we need to coordinate the value of the `default` option with the activated target: I will document that the value of `default` is the last created target.

Reply:
As we have a 1 to 1 connection between websocket session and a tab, I believe we do not see the impact. If we were connected to the browser, it may be "visible". The doc says that activate means bringing the tab to the foreground. With our design, this means nothing - you need to explicitly connect to this new tab with target id to use it. if we were able to use several tab with the same connection, it could be useful - like in [pupetteer](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagebringtofront). 
For now, we can leave it as is, and do not use this endpoint